src/site/content/pl/offline.njk

```diff
@@ -0,0 +1,17 @@
+---
+layout: layout
+title: Offline
+description: |
+  Network Offline
+noindex: true
+offline: true
+---
+
+<div class="w-layout-container--large">
```

Comment:
Should this change to an include that uses the `i18n` frontmatter approach to replace the strings?